assets/src/edit-story/components/library/useInsertElement.js

```diff
@@ -24,34 +24,17 @@ import {
 import { createNewElement, getDefinitionForType } from '../../elements';
```

Comment:
In my PR [here](https://github.com/google/web-stories-wp/pull/376), I moved this file to `edit-story/app/media/actions/useInsertMediaElement.js`